src/SFA.DAS.LevyTransferMatching/Application/Queries/GetPledge/GetPledgeQueryHandler.cs

```diff
@@ -0,0 +1,42 @@
+﻿using MediatR;
+using SFA.DAS.LevyTransferMatching.Interfaces;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.LevyTransferMatching.Application.Queries.GetPledge
+{
+    public class GetPledgeQueryHandler : IRequestHandler<GetPledgeQuery, GetPledgeResult>
```

Comment:
As discussed in web PR, it feels like this should change to be `GetOpportunity`, but we'd still be calling the GetPledge endpoint from the inner api. I guess this is where we map from pledge -> opportunity.

Reply:
This has been actioned. The service that talks to the inner API still refers to "pledge", but that's because the inner API exposes out "pledges" still, not "opportunities".